api/v1alpha1/selfnoderemediationtemplate_types.go

```diff
@@ -85,17 +84,5 @@ func NewRemediationTemplates() []*SelfNodeRemediationTemplate {
 				},
 			},
 		},
-		{
-			ObjectMeta: metav1.ObjectMeta{
```

Comment:
IMHO we need to also check if this one exists in the cluster, and if so delete it?

Reply:
IIUC your aiming for the use case of cleaning leftovers from a previous installation of SNR.
I don't disagree but I don't think it's a blocker, IMO the proper place to do it would be along with other cleanup procedures (old daemonsets, template, config ect).
If you agree I can open a ticket for that.

Reply:
I agree we have more things to clean up after uninstall of SNR, but for this one (the deleted template) I'd prefer to deal with it in this PR. NHC makes a lot more noise when a template isn't found (log, condition, event) than SNR does for an unknown strategy (just a log). I want to make sure users have a good chance to notice that their current config doesn't work anymore. By deleting the default node deletion template we increase that chance significantly IMHO.